config/crd/bases/kepler.system.sustainable.computing.io_keplers.yaml

```diff
@@ -37,6 +37,9 @@ spec:
             properties:
               collector:
                 properties:
+                  collectorPort:
+                    default: 9102
```

Comment:
would this be the default?

Reply:
That's from the previous commit. `make generate` and `make manifests` should have updated this.
I will fix it